keyboards/keebio/iris/keymaps/antonlindstrom/keymap.c

```diff
@@ -0,0 +1,114 @@
+                       LGUI,LOWR,SPC ,         ENT ,RASE,LALT
+  //                  `----+----+----'        `----+----+----'
+  ),
+
+  [_LOWER] = LAYOUT(
+  //,-------+-------+-------+-------+-------+-------.                    ,-------+-------+-------+-------+-------+-------.
+     SE_TILD,KC_EXLM,SE_AT  ,KC_HASH,SE_DLR ,KC_PERC,                     SE_CIRC,SE_AMPR,SE_ASTR,SE_SLSH,SE_LPRN,SE_RPRN,
+  //|-------+-------+-------+-------+-------+-------|                    |-------+-------+-------+-------+-------+-------|
+     SE_ACUT,RGB_TOG,RGB_MOD,RGB_M_B,RGB_M_K,_______,                     _______,_______,_______,_______,_______,SE_ASTR,
+  //|-------+-------+-------+-------+-------+-------|                    |-------+-------+-------+-------+-------+-------|
+     _______,_______,SE_LESS,SE_GRTR,SE_PIPE,SE_SLSH,                     SE_BSLS,SE_LCBR,SE_RCBR,SE_LBRC,SE_RBRC,SE_APOS,
+  //|-------+-------+-------+-------+-------+-------+-------.    ,-------|-------+-------+-------+-------+-------+-------|
+     _______,KC_NUBS,_______,_______,_______,_______,SE_LCBR,     SE_RCBR,_______,_______,_______,_______,_______,_______,
+  //`-------+-------+-------+--+----+-------+-------+-------/    \-------+-------+-------+-------+-------+-------+-------'
+                                   _______,_______,_______,         _______,_______,_______
+  //                              `-------+-------+-------'        `-------+-------+-------'
+  ),
+
+  [_RAISE] = LAYOUT(
+  //,-------+-------+-------+-------+-------+-------.                    ,-------+-------+-------+-------+-------+-------.
+     KC_F12 ,KC_F1  ,KC_F2  ,KC_F3  ,KC_F4  ,KC_F5  ,                     KC_F6  ,KC_F7  ,KC_F8  ,KC_F9  ,KC_F10 ,KC_F11 ,
+  //|-------+-------+-------+-------+-------+-------|                    |-------+-------+-------+-------+-------+-------|
+     SE_GRV ,_______,_______,_______,_______,_______,                     _______,KC_PGUP,KC_UP  ,KC_PGDN,KC_PSCR,KC_PAUS,
+  //|-------+-------+-------+-------+-------+-------|                    |-------+-------+-------+-------+-------+-------|
+     _______,_______,_______,_______,_______,_______,                     KC_HOME,KC_LEFT,KC_DOWN,KC_RGHT,KC_INS ,KC_DEL ,
+  //|-------+-------+-------+-------+-------+-------+-------.    ,-------|-------+-------+-------+-------+-------+-------|
+     _______,_______,_______,_______,_______,_______,KC_VOLU,     KC_VOLD,_______,_______,_______,_______,_______,_______,
+  //`-------+-------+-------+--+----+-------+-------+-------/    \-------+-------+-------+-------+-------+-------+-------'
+                                   _______,_______,_______,         _______,_______,_______
+  //                              `-------+-------+-------'        `-------+-------+-------'
+  )
+};
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
```

Comment:
Could you remove this function?  

It's been deprecated in favor of the `set_single_persistent_default_layer` function (which is a core function).

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer